apps/frontend/src/app/pages/company/gpts/gpts.component.html

```diff
@@ -0,0 +1,347 @@
+<div class="container mx-auto min-h-screen px-3 sm:px-0">
+  <div class="pb-4 pt-6 flex flex-wrap justify-between gap-6">
+    <h2 class="font-sans font-medium text-4xl text-base-100">GPTs</h2>
```

Comment:
![image](https://github.com/Venture-Miner/wagademy/assets/111710515/7c6a4581-54ff-451d-a1db-3577abce094a)

These look different from the figma version

Reply:
Different in what sense? Size? Color? Could you please explain better?

Reply:
Gabriel, I needed to modify some colors for accessibility in the lighthouse to be 100%